image/image.go

```diff
@@ -202,6 +202,18 @@ func bootstrapToRootDir(sto Store, model *asserts.Model, opts *Options) error {
 			return err
 		}
 
+		// fetch the snap assertions too
+		sha3_384, _, err := asserts.SnapFileSHA3_384(fn)
+		ref := &asserts.Ref{
+			Type:       asserts.SnapRevisionType,
+			PrimaryKey: []string{sha3_384},
+		}
+		if err := f.Fetch(ref); err != nil {
+			logger.Noticef("cannot fetch assertion %q: %s", ref.Unique(), err)
```

Comment:
also we really need to decide how print refs to users, this would be the first place to use Unique for that; (we don't have many places doing this actually), but it starts to bother me